fastly/utils.py

```diff
@@ -20,3 +20,23 @@ def purge_url(path):
         return response
 
     return None
+
+
+def purge_surrogate_key(key):
+    """
+    Purge a Fastly.com Surrogate-Key given a key.
+    """
+    if settings.DEBUG:
+        return
+
+    api_key = getattr(settings, 'FASTLY_API_KEY', None)
+    service_id = getattr(settings, 'FASTLY_SERVICE_ID', None)
+    if api_key and service_id:
```

Comment:
should we do response code checking here to catch issues?

Reply:
Maybe worth doing a retry, but in general I'm unsure. It's going to run automated/unsupervised so if we do consider the response code the error handling needs to be automatic.

In general, fastly's API has pretty great availability (we purge hundreds of times a minute for PyPI without seeing incidents often).